packages/quantic/force-app/main/default/lwc/quanticA11yHandler/quanticA11yHandler.js

```diff
@@ -0,0 +1,86 @@
+import {LightningElement} from 'lwc';
+import {isParentOf} from 'c/quanticUtils';
+
+/**
+ * The `QuanticA11yHandler` component handles the accessibility of the Quantic interfaces. It makes the necessary changes to the accessibility tree each time a QuanticResultQuickview or a QuanticRefineModal is opened.
+ * @category Search
+ * @category Insight Panel
+ * @example
+ * <c-quantic-a11y-handler></c-quantic-a11y-handler>
+ */
+export default class QuanticA11yHandler extends LightningElement {
```

Comment:
This is very similar to `atomic-focus-trap`, and I think the name "A11yHandler" is a bit too generic, perhaps this could also be named "focus trap".

Reply:
`QuanticFocusAndNavigationEtcTrapHandler`

Reply:
What about this more specific name:  `QuanticModalA11yHandler`?

Reply:
I assumed we might have more needs to handle a11y-related events in the future and the fact that this component was generic was a good thing. If we scope it to modals only, and later we need to add another handler that isn't modal-related then we'll need to start wrapping a11y handlers within a11y handlers in a certain order and i can see it getting messy. Am I inventing problems? What do you think?

Reply:
Well I don't think that every a11y issue will require a solution based on sending and handling events, so far and amongst all the issues highlighted by deque, the modal problem is the only one requiring this approach to be solved. and generally speaking modals are more or less the main accessibility thing that affects all the other elements in the page which is the reason we used this approach of creating a whole new component that will add aria-hidden to multiple elements in the page to make the modals accessible.
All I'm saying is that I don't think we will need to use this approach to solve a11y issues that are not related to modals.

Reply:
You've been a lot more hands-on than myself for these accessibility fixes so I trust you on that. If `QuanticModalA11yHandler` is what you prefer then go for it 👍